lib/tasks/version.rake

```diff
@@ -47,4 +51,22 @@ namespace :version do
     end
     puts version if verbose
   end
+
+  desc "Show the version"
+  task :show do
+    puts cmake_version
+  end
+
+  desc "Create a git tag with this version"
+  task :tag do
+    create_version_tag { cmake_version }
+    # To use the version number from the .spec file, just use
+    #   create_version_tag
+    # (without the code block)
+  end
+
+  desc "Create a git tag with this version (master branch only)"
+  task :tag_if_master do
+    create_version_tag { cmake_version } if master?
+  end
```

Comment:
well, if all those calls use spec_version instead of cmake_version, then I think it fits perfectly to packaging rake tasks.
What I miss here is automatic creation of tag in case of osc:sr ( which is what jenkins calls ). With current implementation I think it will need manual call, not?

Reply:
IMHO that should remain separate: We have internal and external Jenkins, and we want a well-defined place where we create those tags. We should not leave it to coincidence which Jenkins job is executed first (which would be the one that creates the tag). The second and later calls would just do nothing; or, who knows, in the future we might decide to raise an error if we try to create an already existing tag.

Since this is intended for our external community, we should do that in the _external_ Jenkins only so the version in external OBS matches the tagged source tree.

In an ideal world, this should not matter, but we should minimize the risk for failure here.